AppController/djinn.rb

```diff
@@ -4774,6 +4774,9 @@ def prep_app_dashboard()
       setup_app_dir(AppDashboard::APP_NAME, true)
     }
 
+    # Allow fewer dashboard instances for small deployments.
+    min_dashboards = [3, get_all_appengine_nodes.length].min
```

Comment:
Is it possible to change this to the calculation for how many appservers we're able to run currently minus an arbitrary number (3 would probably work) and then doing min, theoretically leaving room for at least another app to be deployed?